Palo Alto Networks BPA - Lengthen the timeout of the ***pan-os-bpa-sub… (#12022)
demisto/content: Packs/BPA/ReleaseNotes/1_2_5.md

#### Integrations
##### Palo Alto Networks BPA

> - Lengthen the timeout of the ***pan-os-bpa-submit-job*** command.

Author: ```suggestion
- Added the *timeout* argument to the ***pan-os-bpa-submit-job*** command.
```
